src/ui/Typography.tsx

```diff
@@ -1,4 +1,3 @@
-import styled from '@emotion/styled';
 import { FC } from 'react';
 
 const Typography: FC = ({ children }) => {
```

Comment:
i'd be happy to abandon the typography component in this project and just use semantic elements (p, h1... h6) inside specific components

Reply:
so if you agree we can delete this file